### PR TITLE
Fix blur handler on IE where menu ref is undefined

### PR DESCRIPTION
**Pull request by gaastonsr**
On IE when the input is blurred it fails because on isEqualNode does not accept undefined or null values.

This re-introduces the fix on #539 and checks menuDOM is not undefined/null.

### CODE REVIEW

File: src/Select.js [hunk 1]
```diff
@@ -208,9 +208,12 @@ const Select = React.createClass({
 	},
 
 	handleInputBlur (event) {
-		if (document.activeElement.isEqualNode(this.refs.menu)) {
-			return;
-		}
+ 		let menuDOM = ReactDOM.findDOMNode(this.refs.menu);
```
Comment (cloudkite) [inline src/Select.js]: @gaastonsr findDOMNode is unnecessary in react 0.14 as this.refs.menu is a DOM Node. see https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html

Comment (gaastonsr) [inline src/Select.js]: Thanks, I'll update this and check if I can fix #649.

**Comment by gaastonsr** [comment]
I updated the branch with the fixes pointed out by @cloudkite and verified it fixes #649.

**Comment by JedWatson** [comment]
Thanks @gaastonsr! really appreciate this :)

**Comment by gaastonsr** [comment]
Thank you for merging. :)

**Pull request merged**